daemon/qrexec-daemon.c

```diff
@@ -1217,13 +1217,43 @@ static void sanitize_message_from_agent(struct msg_header *untrusted_header)
     }
 }
 
+static bool validate_request_id(struct service_params *untrusted_params, const char *msg)
+{
+    for (size_t i = 0; i < sizeof(untrusted_params->ident); ++i) {
+        switch (untrusted_params->ident[i]) {
+        case '0' ... '9':
+        case 'A' ... 'Z':
+        case 'a' ... 'z':
+        case '_':
+        case '-':
+        case '.':
+        case ' ':
+            continue;
+        case '\0':
+            size_t terminator_offset = i;
```

Comment:
In this PR alone it builds fine, but when merged to master (together with other approved PRs) it fails with:
```
../daemon/qrexec-daemon.c:1235:13: error: expected expression
            size_t terminator_offset = i;
            ^
1 error generated.
```

I'll add `{}` here, not sure why it builds fine without it in just this PR.